python/cuda_parallel/cuda/parallel/experimental/_bindings.pyx

```diff
@@ -0,0 +1,1664 @@
+# distutils: language = c++
+# cython: language_level=3
+# cython: linetrace=True
+
+from libc.string cimport memset, memcpy
+from libc.stdint cimport uint8_t, uint32_t, uint64_t
+from cpython.bytes cimport PyBytes_FromStringAndSize
+
+from cpython.buffer cimport (
+    Py_buffer, PyBUF_SIMPLE, PyBUF_ANY_CONTIGUOUS,
+    PyBuffer_Release, PyObject_CheckBuffer, PyObject_GetBuffer
+)
+
+import ctypes
+
+cdef extern from "<cuda.h>":
+    cdef struct OpaqueCUstream_st
+    cdef struct OpaqueCUkernel_st
+    cdef struct OpaqueCUlibrary_st
+
+    ctypedef int CUresult
+    ctypedef OpaqueCUstream_st *CUstream
+    ctypedef OpaqueCUkernel_st *CUkernel
+    ctypedef OpaqueCUlibrary_st *CUlibrary
+
+
+cdef extern from "cccl/c/types.h":
+    ctypedef enum cy_cccl_type_enum 'cccl_type_enum':
+        cy_CCCL_INT8   'CCCL_INT8'
```

Comment:
Can we drop the `cy_` prefix everywhere here?

Reply:
done